reactDashboard/src/pages/Regional.jsx

```diff
@@ -129,16 +146,15 @@ const Regional = ({
         </Col>
       </Row>
       <Row className="fullscreen-charts">
```

Comment:
Minor point - fullscreen-charts isn't really accurate. We can rename at some point